chicago/views.py

```diff
@@ -338,6 +345,81 @@ class ChicagoCommitteeDetailView(CommitteeDetailView):
 class ChicagoEventsView(EventsView):
     template_name = "events.html"
 
+    def get_context_data(self, **kwargs):
+        context = super(EventsView, self).get_context_data(**kwargs)
+
+        # Get year range for datepicker.
+        aggregates = ChicagoEvent.objects.aggregate(
```

Comment:
i assume this is mainly copy/paste from the django-councilmatic model.

this could be improved a good bit by *not* using the parameter, but instead defining two url rules

the current:
```python
url(r"^events/$", views.EventsView.as_view(), name="events")
```
which would dispatch to the default upcoming events and

```
url(r"^events/(?P<date>\d{4}-{2}-{2})/$", views.MonthEventsView.as_view(), name="events"),
```

which dispatched to new view for just that selected month.

this is not a required change. if you don't feel like doing this now, we can open up an issue and do it later.

Reply:
good call - lets do this as part of another PR